### DISCUSSION TITLE
basic-extra-meal/jdk9-char-byte-string

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#116] [Announcements]
# basic-extra-meal/jdk9-char-byte-string

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，jdk9为什么要将String的底层实现由char数组改成了byte数组?

https://tobebetterjavaer.com/basic-extra-meal/jdk9-char-byte-string.html

**Comment by GongNanyue** [comment]
在此之前呢，中文是两个字节，应为也是两个字节
改成
英文也是两个字节
